src/ui/layout/container.svelte

```diff
@@ -54,12 +53,14 @@
     position: fixed;
     width: 100vw;
     min-height: 100vh;
+    z-index: 1000001;
```

Comment:
@nicfix changed your logic of selecting `inside` or `fullscreen` classes as per my way when merging (we were doing almost the same in out PRs)